src/components/CodeiumEditor/CompletionProvider.ts

```diff
@@ -64,11 +64,11 @@ export class MonacoCompletionProvider {
 
   ) {
     this.metadata = new Metadata({
-      ideName: getBrowserVersion() ?? "unknown",
-      ideVersion: getCurrentURL() ?? "unknown",
+      ideName: "web",
+      ideVersion: getBrowserVersion(),
```

Comment:
Can we keep the browser URL as the IDE version? Would like to keep  that attribution ideally